plugins/woocommerce/src/Admin/API/Reports/Orders/DataStore.php

```diff
@@ -612,9 +618,7 @@ protected function get_order_attributions_by_order_ids( $order_ids ) {
 	public static function get_all_statuses() {
 		global $wpdb;
 
-		$cache_key = 'orders-all-statuses';
-		$statuses  = Cache::get( $cache_key );
-
+		$statuses = get_transient( 'woocommerce_analytics_orders_statuses_all' );
```

Comment:
Any particular reason we're shifting over to manual implementation of the cache over the `Cache` class for reports?

Reply:
Yes, it's not allowing us to set a custom retention period, and I want to avoid touching any public APIs in this PR.

Reply:
That makes sense.  Thanks for the explanation!